src/nlmdb.cc

```diff
@@ -9,17 +9,17 @@
 
 namespace nlmdb {
 
-void Init (v8::Handle<v8::Object> target) {
+NAN_MODULE_INIT(init) {
   Database::Init();
   WriteBatch::Init();
   Iterator::Init();
 
   v8::Local<v8::Function> nlmdb =
-      v8::FunctionTemplate::New(NLMDB)->GetFunction();
+      Nan::New<v8::FunctionTemplate>(NLMDB)->GetFunction();
 
-  target->Set(v8::String::NewSymbol("nlmdb"), nlmdb);
+  target->Set(Nan::New("nlmdb").ToLocalChecked(), nlmdb);
 }
 
-NODE_MODULE(nlmdb, Init)
```

Comment:
fwiw name isn't important here, could just as well be `Init` and you could also avoid `NAN_MODULE_INIT` by changing the `Handle` to a `Local` in the original `Init` declaration. No big deal either way though.